tests/ui/extenv/extenv-env-overload.rs

```diff
@@ -0,0 +1,8 @@
+// run-pass
+// compile-flags: --env PATH=123abc -Zunstable-options
```

Comment:
can you also use `// rustc-env:` here and not use `PATH`?

Reply:
I resolved this too early, thinking that Urgau already commented on it. Can you address it?

Reply:
Ah indeed. Updated it.